tests/io/test_empad.py

```diff
@@ -124,15 +128,19 @@ def test_sum_analysis(default_empad, lt_ctx):
     assert np.count_nonzero(results[0].raw_data) > 0
 
 
-def test_pick_job(default_empad, lt_ctx):
+def test_pick_job(default_empad, lt_ctx, TYPE):
```

Comment:
Missing the parametrize mark here?

Reply:
Actually, added the TYPE in error since this is using the JOB directly. No UDF here!

Nice catch, thx!